modules/cluster-logging-collector-rsyslog.adoc

```diff
@@ -0,0 +1,172 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-rsyslog_{context}"]
+= Configuring the Fluentd remose-syslog plug-in to send logs to an external log aggregator
+
+You can configure Fluentd to send a copy of its logs to an external Rsyslog server, 
+and not the default Elasticsearch, using the *fluent-plugin-remote-syslog* plug-in.
+
+[NOTE]
+====
+This legacy *fluent-plugin-remote-syslog* method is deprecated and will be removed in a future release.
```

Comment:
But I thought the plan was to use a different plugin which will support TLS and RFC5424?